partEZ/database/migrations/2016_01_27_004712_create_poll_options_table.php

```diff
@@ -15,13 +15,15 @@ public function up()
         if (!Schema::hasTable('poll_options'))
         {
             Schema::create('poll_options', function (Blueprint $table) {
-                $table->increments('oid')->unsigned();
+                $table->increments('oid');
                 $table->integer('pid')->unsigned();
                 $table->string('option');
-                $table->primary(['oid', 'pid']);
+                // $table->primary(array('oid', 'pid'));
```

Comment:
We should probably drop and recreate the table in a separate migration so that we don't have to delete everything in  our migrations table